onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportProblemFragmentBase.java

```diff
@@ -55,6 +55,13 @@
      */
     GoogleApiClient mGoogleApiClient;
 
+    /**
```

Comment:
I wouldn't define this as an abstract method in the base class.  This method isn't used in the base class, so it should just be implemented as a normal method in the subclasses.